### PR TITLE
fix: Show the full URL at the end of an Enterprise run

### PR DESCRIPTION
**Pull request by guilgaly**
It only showed the path.

-----

It might make more sense for the `SimulationStartResult` to contain the full URL (it currently contains only the path, and we then have to reconstruct the full URL to display a link to the user). There would be impacts on all the plugins though, so not doing it right now.

### CODE REVIEW

**Review by slandelle — APPROVED** [review]

File: src/main/java/io/gatling/plugin/client/http/HttpEnterpriseClient.java [hunk 1]
```diff
@@ -65,6 +67,11 @@ public HttpEnterpriseClient(URL baseUrl, String token, String client, String ver
     new PrivateApiRequests(publicApiBaseUrl, token).checkVersionSupport(client, version);
   }
 
+  @Override
+  public URL getBaseUrl() {
+    return baseUrl;
```
Comment (Isammoc) [inline src/main/java/io/gatling/plugin/client/http/HttpEnterpriseClient.java:72]: If URL is mutable, we should return a copy.

May be of importance for shared client but for different servers (as we allow now for Jenkins for instance)

**Review by Isammoc — COMMENTED** [review]
`SimulationStartResult` won't contain the full url.

The main reason for that is that server cannot be sure what is the base url for each client (different proxies in a multi networks environment)

File: src/main/java/io/gatling/plugin/client/http/HttpEnterpriseClient.java (hunk 1 above)
Comment (slandelle) [reply src/main/java/io/gatling/plugin/client/http/HttpEnterpriseClient.java:72]: AFAIK, URL is immutable.

**Review by Isammoc — APPROVED** [review]

**Comment by guilgaly** [comment]
> `SimulationStartResult` won't contain the full url.
> The main reason for that is that server cannot be sure what is the base url for each client (different proxies in a multi networks environment)

I'm not talking about doing it server-side. `SimulationStartResult` is built in the client, aggregating data resulting from several API calls and from the user's choice to create a new simulation or use an existing one.

**Comment by slandelle** [comment]
@guilgaly Release 1.5.1 and update other PRs?

**Comment by guilgaly** [comment]
@slandelle need to finish another PR to add private locations to the list when configuring a new simulation on Gatling Enterprise, I'll do a new release when that's ready too.

**Pull request merged**